src/js/user-profile/components/UserDataSection.jsx

```diff
@@ -8,28 +8,24 @@ import moment from 'moment';
 class UserDataSection extends React.Component {
   render() {
     let content;
-    const name = `${this.props.name.first} ${this.props.name.middle} ${this.props.name.last}`;
+    const name = `${this.props.name.first ? this.props.name.first : ''} ${this.props.name.middle ? this.props.name.middle : ''} ${this.props.name.last ? this.props.name.last : ''}`;
```

Comment:
@crwallace you can consider using `compact` here to avoid all the ternary statements. I had to deal with null/falsey values for facility addresses too.

something like this  `compact([first, middle, last]).join(' ')`